website/public/blog/blog-head.ejs.html

```diff
@@ -1,7 +1,15 @@
 <%#
 <!-- Copyright (C) 2020  Matthew "strager" Glazar -->
 <!-- See end of file for extended copyright information. -->
-%>
+%> <% if (typeof meta.blogDate !== 'undefined') { %>
+<meta property="og:type" content="article" />
+<meta property="article:section" content="Technology" />
+<meta property="article:published_time" content="<%= meta.blogDate %>" />
```

Comment:
We know this is a blog article because the file is called blog-head.ejs.html. Shouldn't we only put the `if` around `article:published_time`?
```suggestion
%>
<meta property="og:type" content="article" />
<meta property="article:section" content="Technology" />
<% if (typeof meta.blogDate !== 'undefined') { %>
<meta property="article:published_time" content="<%= meta.blogDate %>" />
```

Reply:
This file is also included in /blog.